Install Issue: lodash - node version

Hi all,
I'm getting the following issue trying to install sass-lint.

```
No compatible version found: lodash.upperfirst@^3.0.0
npm ERR! Valid install targets:
npm ERR! 4.1.2, 4.1.1, 4.1.0, 4.0.2, 4.0.1, 4.0.0

```
I'm running node 4.3.1 and npm 3.75. 

I assume this means that lodash.upperfirst 3.0.0 doesn't support the version of node I'm running, is there anyway round this, other than downgrading my node version (which I don't want to do).

Best,
Max

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "glob": "^6.0.0",
     "gonzales-pe": "3.0.0-31",
     "js-yaml": "^3.2.6",
-    "lodash.capitalize": "^3.0.0",
+    "lodash.capitalize": "3.0.0",
     "lodash.kebabcase": "^3.0.1",
     "merge": "^1.2.0",
     "util": "^0.10.3"

